client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -218,10 +218,10 @@ export const isCustomsFormRequired = createSelector(
  * @param {Object}  appState            Local Redux state.
  * @param {Object}  addressData         Address to check, including normalization state and values.
  * @param {number}  siteId              The ID of the current site ID.
- * @param {boolean} shouldValidatePhone An indiator whether phone validation is required.
+ * @param {Object}  fieldsToValidate    Specify which form fields to validate.
  * @return {Object}                     A hash of errors with field names as keys.
  */
-const getRawAddressErrors = ( appState, addressData, siteId, shouldValidatePhone ) => {
+ export const getRawAddressErrors = ( appState, addressData, siteId, fieldsToValidate ) => {
```

Comment:
When I set a break point on the calls to `getRawAddressErrors()`, it seems like `fieldsToValidate` is a boolean than an object.